tpm/tpm.go

```diff
@@ -561,9 +567,9 @@ func MakeIdentity(rw io.ReadWriter, srkAuth []byte, ownerAuth []byte, aikAuth []
 			return nil, err
 		}
 
-		// We can't pack the pair of values directly, since the label is
+		// We can't tpmutil.Pack the pair of values directly, since the label is
```

Comment:
This looks like another copy-paste error. I think the comment should probably just say "pack", not "tpmutil.Pack"